docs/modules/ROOT/nav.adoc

```diff
@@ -1,13 +1,16 @@
 include::user:ROOT:partial$shared-nav.adoc[]
 
 .About
-* https://products.docs.vshn.ch/products/appuio/cloud/index.html[Product Description 🔗^]
+* https://status.appuio.cloud[Status 🔗^]
```

Comment:
Can we add the external link indication to the theme? Having to do it like this is guaranteed to be missed by content authors.